rasr/feature_scorer.py

```diff
@@ -126,25 +127,50 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        mixtures,
+        model,
+        io_map,
+        *args,
```

Comment:
*args can be removed, the `FeatureScorer` anyway does not have any extra parameters. In addition, it would maybe be good to place a `*` after self to enforce all parameters to be passed as kwargs.